### DISCUSSION TITLE
Lighthouse V1 base

### DISCUSSION THREAD

**Discussion by ddwy** [bitcraze/discussions#1228] [Q&A]
Hello, I am currently using Lighthouse V1 base station to test Lighthouse Positioning. What does the button pressed here mean when Manage geometry reaches the fifth step？
![微信截图_20240312104413](https://github.com/bitcraze/crazyflie-clients-python/assets/43903311/0cb09484-c3fc-467b-96bf-0649a88f68bb)

**Comment by knmcguire** [comment]
You are referring to the 'Estimate Geometry' button? That will calculate the location and orientation of the basestation based on the data it has recorded so far. You don't need to do anything else than push the button and wait if there is a result. 

Please check out the tutorial if you haven't found it already, as that has a video of how to go through the wizard https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/
